job/scriptsGen/generateKillScript.js

```diff
@@ -0,0 +1,100 @@
+'use strict';
+
+var self = generateKillScript;
+module.exports = self;
+
+var path = require('path');
+var fs = require('fs-extra');
+
+var generateScriptFromTemplate = require('./generateScriptFromTemplate.js');
+
+function generateKillScript(externalBag, callback) {
+  var bag = {
+    killScriptTemplatePath:
+      path.join(global.config.execTemplatesPath, 'job', 'kill_task.sh'),
+    killScriptPath: path.join(externalBag.buildScriptsDir, 'kill_task.sh'),
```

Comment:
This is a deviation from the current design. Instead of saving the script in the reqKick directory, it gets saved in the build/scripts directory which I think is more appropriate as it gets cleaned up after every build.